Resources/Locale/en-US/cargo/cargo-console-component.ftl

```diff
@@ -9,18 +12,22 @@ cargo-console-menu-points-label = Balance:{" "}
 cargo-console-menu-points-amount = ${$amount}
 cargo-console-menu-shuttle-status-label = Shuttle status:{" "}
 cargo-console-menu-shuttle-status-away-text = Away
+cargo-console-menu-shuttle-status-capacity = {$currentCoutn}/{$maxCapacity}
```

Comment:
this appears to be unused (and also has a typo if you were planning to use it)

Reply:
Thanks, deleted it